src/token/api/src/account.rs

```diff
@@ -1,21 +1,24 @@
 use std::fmt::{Display, Formatter};
 
 use candid::{CandidType, Principal};
-use serde::{Deserialize, Serialize};
+use serde::Deserialize;
 
 use crate::error::TxError;
 
 pub static DEFAULT_SUBACCOUNT: Subaccount = [0u8; 32];
 
-#[derive(Debug, Clone, CandidType, Deserialize, Copy, PartialEq, Eq, Serialize)]
+#[derive(Debug, Clone, CandidType, Deserialize, Copy, PartialEq, Eq)]
 pub struct Account {
     pub owner: Principal,
-    pub subaccount: Option<Subaccount>,
+    pub subaccount: Subaccount,
```

Comment:
why is subaccount not an opt as in the did file for the ICRC-1 spec? https://github.com/dfinity/ICRC-1/blob/main/ICRC-1.did